src/fal/cli/flow_runner.py

```diff
@@ -95,3 +108,55 @@ def _unique_id_to_model_name(unique_id: str):
 
 def _unique_ids_to_model_names(id_list: List[str]):
     return list(map(lambda id: _unique_id_to_model_name(id), id_list))
+
+
+def _combine_fal_run_results(target_path):
+    last_run_result_file = os.path.join(target_path, "run_results.json")
+    last_run_result = _get_all_result_content(last_run_result_file)
+    last_run_result_array: List[str] = last_run_result["results"]
+    all_results = _distinct_by(
+        reduce(
+            lambda all, next: all.extend(_get_result_array(next)) or all,
+            _get_fal_run_results(target_path),
+            last_run_result_array,
+        ),
+        "unique_id",
+    )
+    last_run_result.pop("results")
+    last_run_result["results"] = all_results
+    os.remove(last_run_result_file)
+    with open(last_run_result_file, "w") as file:
+        file.write(json.dumps(last_run_result))
+        file.close()
+
+
+def _distinct_by(target_list, key):
+    seen_keys = []
+    result = []
+    for item in target_list:
+        if item[key] not in seen_keys:
+            seen_keys.append(item[key])
+            result.append(item)
```

Comment:
interesting what we are giving priority to here. We are keeping the OLDEST `fal_results_X` values.

I have two options when `item[key] in seen_keys`

1. We should or keep the NEWEST value
2. We should raise an exception because we should not run the same unique id twice.

Reply:
as discussed we dont even need this